scripts/run_tests.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-#
-# Run project tests
-#
-# NOTE: This script expects to be run from the project root with
-# ./scripts/run_tests.sh
-
-make test
```

Comment:
Hurray for getting rid of this 🔥

Reply:
Why? This is how I generally run my tests. I guess someone needs to force me into the future of using `make`, I still don't really see the value in it TBH.

Reply:
But surely... since it's the *script* running *make*, it's the *script* you don't see value in...?

Reply:
@TheDoubleK would much rather be using Gradle right :-D

Reply:
The script used to be the only way to run things - there was no Makefile.  Some bright spark made the Makefile do what the script used to do, and updated the script to just run make, which meant I never really updated my workflow to use the new Makefile directly.  It's fine.  We can kill this.  I can move on.

Reply:
FWIW I generally run the tests using the `py.test` command because there's a load of useful params you can pass in.

py.test 
-s (print standard out/ print statements as encountered instead of at the end of the test run, also allows pdb in tests) 
-k <name of a single test or test class if you like>
--tb native (for normal python tracebacks)
<name of a file>

Its awesome.

Reply:
Yeah I only dig in & run `py.test` when I want to do proper digging. Most the time I want to run what jenkins'll run (include the QA metrics and javascripty stuff)

Reply:
Yeah I do get caught out from time to time because py.test will pass then I'll push something up then flake8 will fail on jenkins :grin: